App/Lettuce/Sources/Lettuce/MessageCenter/Requests/messaging/message-attachments/DownloadAttchmentRequest.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+import HTTPTypes
+
+import Onion
+
+//https://developer.allegro.pl/documentation#operation/uploadAttachmentPUT
+public struct DownloadAttchmentRequest {
+
+    let attachmentId: Identifier<Attachment>
+
+    var path: String { "/messaging/message-attachments/\(attachmentId.id)" }
+
+    var method: HTTPRequest.Method { .get }
```

Comment:
Documentation says "PUT" :)

Reply:
but task is about get :P  probably error with copying link from allegro.
<img width="1431" alt="Screenshot 2023-11-28 at 20 50 59" src="https://github.com/sloik/seller/assets/22295493/800b70ae-e580-4db0-a61b-5f4425117f9a">